src/context.rs

```diff
@@ -40,4 +41,10 @@ pub struct Context {
     pub app_name: String,
     #[serde(default)]
     pub environment: String,
+    #[serde(default = "current_time", rename = "currentTime")]
+    pub current_time: String,
```

Comment:
In fact, I'm not sure whether we should be capturing the current time at all. For the proxy use case, clients should not be permitted to time-travel; for the non-proxy use case access to the current time is probably cheap enough, since time access is such a well known thing.

We could compile rules that relate to get compiled into a unix time expression for even faster evaluation, if scaling the proxy is required - though I suspect only unleash themselves would hit that scale.